### PR TITLE
AA-137: Add first-section celebration

### PR DESCRIPTION
**Pull request by mikix**
When a learner completes their first section in a course, throw up a modal that celebrates that fact and encourages them to share progress.

https://openedx.atlassian.net/browse/AA-137

Screenshot from 2020-06-12 11-40-20

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!doctype html>
 <html lang="en-us">
   <head>
-      <title>Course | edX</title>
+      <title>Course | <%= process.env.SITE_NAME %></title>
```
Comment (mikix) [inline public/index.html:4]: Unrelated, but threw it in for free.

File: src/courseware/course/celebration/utils.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import { sendTrackEvent } from '@edx/frontend-platform/analytics';
+import { getAuthenticatedUser } from '@edx/frontend-platform/auth';
+
+import { setFirstSectionCelebrationComplete } from '../../../data/api';
+import { clearLocalStorage, getLocalStorage, setLocalStorage } from '../../../data/local';
+
+const CELEBRATION_LOCAL_STORAGE_KEY = 'CelebrationModal.showOnSectionLoad';
+
+// Records clicks through the end of a section, so that we can know whether we should celebrate when we finish loading
+function handleNextSectionCelebration(courseId, sequenceId, nextSequenceId) {
```
Comment (Dillon-Dumesnil) [inline src/courseware/course/celebration/utils.jsx]: Why does this take in the `courseId`?

File: src/data/local.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+// This file holds some convenience methods for dealing with localStorage.
```
Comment (Dillon-Dumesnil) [inline src/data/local.js]: Nit: Rename this file to localStorage.js . Local feels too dev environment-y to me.

**Review by Dillon-Dumesnil — APPROVED** [review]
MFE side looks good!

**Comment by mikix** [comment]
Tagging @Ahernkb and @hilaryg123 to look at screenshot and confirm it's not crazy

File: src/courseware/course/celebration/utils.jsx (hunk 2 above)
Comment (mikix) [reply src/courseware/course/celebration/utils.jsx]: Great catch. An earlier version used it. Removing...

File: src/data/local.js (hunk 3 above)
Comment (mikix) [reply src/data/local.js]: Fair

File: src/courseware/course/celebration/utils.jsx (hunk 2 above)
Comment (mikix) [reply src/courseware/course/celebration/utils.jsx]: I would have expected the linter to catch that.

Comment (Dillon-Dumesnil) [reply src/courseware/course/celebration/utils.jsx]: Yeah, same /shrug

File: public/index.html (hunk 1 above)
Comment (davidjoy) [reply public/index.html:4]: Total FYI - I think we may start using react-helmet in the MFE.

**Review by davidjoy — APPROVED** [review]
Nice refactoring of the localStorage stuff. 👍  The rest looks great too.

**Pull request merged**